app/components/public/index/header.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@ember/component';
+
+export default class Header extends Component {
+  images = ['/images/public/banner.jpg'];
+
+  get headerImageUrl() {
+    return this.images[Math.floor(Math.random() * this.images.length)];
```

Comment:
@Ellen-Wittingen this is not used anywhere. Was this functionality finished? I presume it is meant to randomly select from a bunch of 'kamer-forum banners'?